Libraries/StyleSheet/ColorValueTypes.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict-local
+ */
+
+'use strict';
+
+import type {NativeColorValue} from './PlatformColorValueTypes';
+import {
+  PlatformColor,
+  normalizeColorObject,
+  processColorObject,
+} from './PlatformColorValueTypes';
+
+export type {NativeColorValue};
+export {PlatformColor, normalizeColorObject, processColorObject};
+
+export type ColorValue = null | string | NativeColorValue;
+
+export type ProcessedColorValue = number | NativeColorValue;
```

Comment:
This was nullable in the previous PR, did it change here intentionally?

Reply:
I think its cleaner to not have the '?' here since in the instances that use the type and need to be nullable use ?ProcessedColorValue